bin/build

```diff
@@ -99,7 +101,8 @@ boundary_position_ids = boundary_data.popolo_areas.map { |a| a[:associated_wikid
       membership_rows.select { |m| m[:district] }.map do |m|
         area_wikidata_id = m[:district].value
         unless known_areas.include?(area_wikidata_id)
-          puts "WARNING: the district #{wikidata_labels.item_with_label(area_wikidata_id)} wasn't found in the boundary data for position #{wikidata_labels.item_with_label(m[:role]&.value)}"
+          puts "WARNING: the district #{wikidata_labels.item_with_label(area_wikidata_id)} " \
+               "wasn't found in the boundary data for position #{wikidata_labels.item_with_label(m[:role]&.value)}"
```

Comment:
rather than splitting these like this, perhaps it might be cleaner/clearer to remove the interpolation instead, and go for something more like:

```ruby
    puts "WARNING: the district %s wasn't found in the boundary data for position %s" %
      [wikidata_labels.item_with_label(area_wikidata_id), m[:role]&.value]
```

Reply:
If we were going to use `%` instead, I'd prefer to use named interpolators in the format strings, e.g.

```ruby
        puts "WARNING: the district %<district>s wasn't found in the boundary data for position %<position>s" %
             { district: wikidata_labels.item_with_label(area_wikidata_id),
               position: wikidata_labels.item_with_label(m[:role]&.value), }
```

Would you be OK with that?

Reply:
Definitely. I don't think it's very common in Ruby code, but I think it's very clear (and is a nice step towards abstracting these away even further later should we choose to)

Reply:
(I pushed a fixup commit with changes along those lines.)